iai_semantic_maps/owl/kitchen.owl

```diff
@@ -917,6 +917,16 @@
         <knowrob:hasVisual rdf:datatype="&xsd;boolean">false</knowrob:hasVisual>
     </owl:NamedIndividual>
 
+    <owl:NamedIndividual rdf:about="&iai-map;iai_kitchen_meal_table_counter_top">
+        <rdf:type rdf:resource="&knowrob;CounterTop"/>
+        <knowrob:depthOfObject rdf:datatype="&xsd;double">2.45</knowrob:depthOfObject>
+        <knowrob:heightOfObject rdf:datatype="&xsd;double">0.02</knowrob:heightOfObject>
+        <knowrob:widthOfObject rdf:datatype="&xsd;double">0.8</knowrob:widthOfObject>
```

Comment:
Unfortunately, I cannot find the correct coordinates as our world initial frame is not a physical point. The distance to the kitchen island edge is 126.5 cm and the distance to the fridge bottom edge is 204 cm. Maybe @bbrieber or @fairlight1337 can thus infer the actual pose in Blender or something?